main.go

```diff
@@ -155,21 +155,26 @@ func main() {
 	Listen(port)
 	fmt.Println("Listening on port " + port + " - http://localhost:" + port + "/\n")
 
-	// TODO: Workaround: this initial call is needed only to populate RateLmitSpecs
-	buildGraph(-1)
+	GetRateLimits(token)
 
 	var duration time.Duration
 	if delay != (60 * pages) {
 		duration = time.Second * time.Duration(delay)
 	} else {
+		if reflect.TypeOf(RateLimitSpecs.PollInterval).String() != "int" || RateLimitSpecs.PollInterval <= 0 {
```

Comment:
_I might have to do some research on this, as I guess only the following condition is needed - this first one might always be true._